.github/workflows/ci.yaml

```diff
@@ -0,0 +1,83 @@
+name: CI
+on: [push, pull_request]
+
+jobs:
+  #clang-format:
+  #  runs-on: ubuntu-latest
+  #  steps:
+  #  - uses: actions/checkout@v2
+  #  - uses: DoozyX/clang-format-lint-action@v0.12
+  #    with:
+  #      exclude: './thirdparty'
+  #      clangFormatVersion: 12
```

Comment:
When applying clang-format there is a huge diff. I guess we leave this for now. We also need to settle on a clang-format version and probably update the `.clang-format` file as well.

Reply:
I'd leave this out for now, formatting changes are irrelevant if the project is in life-support mode.